### PR TITLE
feat: silence slack messages during quiet period

### PR DESCRIPTION
**Pull request by erickzhao**
Doesn't post to Slack during December.

~~For individual label additions, I think this change depends on Octokit's `pull_request.updated_at` prop to also reflect the latest time a label was added, since webhook payloads don't seem to come with a timestamp. (Alternatively, we could check the default `new Date()` constructor call similar to how it's done for the reminders.)~~ This now uses `new Date()` for everything.

### CODE REVIEW

**Comment by samuelmaddock** [comment]
This is the message for newly posted APIs, right? I'd personally like to still see the messages since they only get posted when they're initially created.

I think a compromise could be:
- Don't @ the wg-api team during December
- Disable the reminder listing all open PRs during December

File: src/remind.ts [hunk 1]
```diff
@@ -18,7 +18,9 @@ async function main() {
     q: `repo:electron/electron ${q}`,
     sort: 'created',
   });
-  if (items.length) {
+
+  // silence during quiet period
+  if (items.length && new Date().getMonth() < 11) {
```
Comment (nornagon) [inline src/remind.ts:23]: maybe pull this check out into a common `isQuietPeriod()` function?

**Review by nornagon — APPROVED** [review]

**Comment by erickzhao** [comment]
> This is the message for newly posted APIs, right? I'd personally like to still see the messages since they only get posted when they're initially created.

@samuelmaddock Currently it's both of them! I can disable the ping only as a compromise and keep new PR messages coming in :)

File: src/remind.ts (hunk 1 above)
Comment (erickzhao) [reply src/remind.ts]: [3f84def](https://github.com/electron/zoilist/pull/17/commits/3f84def92d404630ffdbcd74de106c582523b6ce)

**Pull request merged**